### PR TITLE
HW-7 was done

### PR DESCRIPTION
**Pull request by rudendan**

### CODE REVIEW

File: HomeWork-7/News/src/main/java/com/example/news/NewsApplication.java [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
+@EnableScheduling
```
Comment (kartius) [inline HomeWork-7/News/src/main/java/com/example/news/NewsApplication.java]: Why did you use this annotation?

File: HomeWork-7/CustomNews/src/main/java/com/example/customnews/service/EndService.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+
+@Service
+public class EndService {
+
+    @Autowired
+    private RestTemplate restTemplate;
+    private final String url = "http://localhost:8080/api/customnews";
```
Comment (kartius) [inline HomeWork-7/CustomNews/src/main/java/com/example/customnews/service/EndService.java]: Better, move it to properties

**Review by kartius — APPROVED** [review]

File: HomeWork-7/News/src/main/java/com/example/news/NewsApplication.java (hunk 1 above)
Comment (rudendan) [reply HomeWork-7/News/src/main/java/com/example/news/NewsApplication.java]: This was for a test "scheduling", forgot to delete it. Deleted

File: HomeWork-7/CustomNews/src/main/java/com/example/customnews/service/EndService.java (hunk 2 above)
Comment (rudendan) [reply HomeWork-7/CustomNews/src/main/java/com/example/customnews/service/EndService.java]: Ok, thanks, done it

**Pull request merged**